packages/fast-form-generator-react/src/form/form-item.children.style.ts

```diff
@@ -1,113 +1,105 @@
-import { ellipsis, toPx } from "@microsoft/fast-jss-utilities";
+import { ellipsis, focusVisible, toPx } from "@microsoft/fast-jss-utilities";
 import {
-    applyAddItemStyle,
     applyAriaHiddenStyles,
     applyCleanListStyle,
+    applyControl,
     applyGlobalStyle,
-    applyHeaderStyle,
     applyInputStyle,
-    applyListItemStyle,
-    applyPopupHeadingStyles,
-    applyPopupMenuStyles,
+    applyLabelStyle,
+    applyRemoveItemStyle,
+    applySoftRemove,
     colors,
-    localizePadding,
-    thickLine,
-    trashcan,
+    insetStrongBoxShadow,
+    rightArrow,
 } from "../utilities/form-input.style";
 import { ComponentStyles, CSSRules } from "@microsoft/fast-jss-manager";
 import { FormItemChildrenClassNameContract } from "../class-name-contracts/";
 
 const styles: ComponentStyles<FormItemChildrenClassNameContract, {}> = {
     "@global": {
         ...applyGlobalStyle(),
+        'body > div > li[draggable="true"] button': {
```

Comment:
this seems a little dangerous - is there not a way to make a more explicit selector?